.github/ISSUE_TEMPLATE/project.md

```diff
@@ -21,35 +21,56 @@ assignees: ''
 
 <!-- Please fill out the following questions to rationalize the prioritization of this project -->
 
-- Does this serve our mission?
+- Does this serve our mission
 
 - Will this help to make us financially sustainable within a year?
 
-- Is it feasible to build and maintain with our team and runway?
-
 - Can this make us move faster?
 
-- Does this increase quality and not technical debt?
+- Does this increase quality? 
+
+- Does this add technical debt?
+
+- Is it feasible to build and maintain with our current team and runway?
 
-- Will this bring us more Collective and financial contributors?
+- Is this problem already solved elsewhere or is it based on insight or research?
 
-- Is this already solved elsewhere or is it our unique contribution?
+- How will we measure our success? Do we have a baseline today that we can compare this against?
 
-## Roadmap & Timeframe
-(Key milestones or phases and estimated delivery dates, including specs, design, feature development, feedback, marketing/comms, and documentation)
+- How well do we understand the complexity of the problem, feature to be added or user need to be addressed? 
 
-**Kick-off meeting**:
+- What is the estimated timeframe for delivering this project? 
 
-## Success metrics
+## Kick-off meeting
+
+### Roadmap & Timeframe
+(Key milestones or phases and estimated dates. Consider including further scoping, discovery, and prototyping. Include as a minimum: planning, design, development, documentationm, feedback, iteration and, marketing )
+
+### Success metrics
 (What we will judge outcomes of the project by)
 
+### Release planning
+
+** Estimated delivery date**
+
+- Which aspect(s) of the platform does this change impact?
```

Comment:
Some of my goals here are also to ensure that we're able to plan documentation edits and improvements